wpilibNewCommands/src/main/native/cpp/frc2/command/ProxyCommand.cpp

```diff
@@ -17,7 +17,9 @@ ProxyCommand::ProxyCommand(Command* command)
 }
 
 ProxyCommand::ProxyCommand(std::unique_ptr<Command> command)
-    : m_supplier([command = std::move(command)] { return command.get(); }) {}
+    : m_supplier((
+          SetName(std::string{"Proxy("}.append(command->GetName()).append(")")),
```

Comment:
This feels hacky. Is there no other way to do it?

Reply:
I suppose initialization of `m_supplier` could be moved to the constructor body